cmd/admin-trace.go

```diff
@@ -171,6 +171,7 @@ type callStats struct {
 	Rx       int           `json:"rx"`
 	Tx       int           `json:"tx"`
 	Duration time.Duration `json:"duration"`
+	Ttfb     time.Duration `json:"timetofirstbyte"`
```

Comment:
It is usually helpful to have camel case or snake case for multi-word JSON keys. It simplifies consuming JSON values of this type in languages other than Go.

Reply:
good suggestion - changed the rest of the multi word keys to camel case also.